src/util/index.js

```diff
@@ -33,6 +36,47 @@ export function buildContent(options : ContentConfig) : string {
     return content || '';
 }
 
+export function getBuyerCountry(fundingOption : string) : string {
```

Comment:
better rename the function to something like `getBuyerrCountryFromFundingSource`